test/js/index.js

```diff
@@ -14,8 +14,11 @@ describe("js", () => {
 			throw new Error("Forgot to remove `solo: true` from test");
 		}
 
-		(solo ? it.only : it)(dir, () => {
-			dir = path.resolve(`${__dirname}/samples`, dir);
+		dir = path.resolve(`${__dirname}/samples`, dir);
+
+		const skip = !fs.existsSync(`${dir}/input.svelte`);
```

Comment:
Is there a particular reason you want to specifically note this as a skipped test? (And the same thing below with the other types of tests.) If there's no `input.svelte`/etc. file, I think we'd just want to return early from the loop. If I switch to a branch with fewer tests, I don't think it should be considered _skipping_ those tests - they just shouldn't exist.

Reply:
Just in case someone typo the input.svelte file name, at least he sees the test got skipped instead of mysteriously not picked up? 🤔

Reply:
Maybe instead of it.skip, I can console out explanation and early return the loop?

Reply:
I think that's the right solution — added